### PR TITLE
Checkout abseil in the smoketest

### PR DESCRIPTION
**Pull request by s-perron**
Now that shaderc knows about abseil, we want to check it out. Hopefully
this fixes the smoketest.

### CODE REVIEW

File: kokoro/scripts/linux/build-docker.sh [hunk 1]
```diff
@@ -131,6 +131,7 @@ elif [ $TOOL = "cmake-smoketest" ]; then
   git clone https://github.com/KhronosGroup/SPIRV-Headers.git spirv-headers
   git clone https://github.com/google/re2
   git clone https://github.com/google/effcee
+  git clone https://github.com/abseil/abseil-cpp abseil_cpp
```
Comment (ben-clayton) [inline kokoro/scripts/linux/build-docker.sh:134]: Should we be pinning these to particular revisions?

**Review by ben-clayton — APPROVED** [review]
Approval, but a question about the checkouts

File: kokoro/scripts/linux/build-docker.sh (hunk 1 above)
Comment (s-perron) [reply kokoro/scripts/linux/build-docker.sh:134]: The smoke test is specifically about checking out the latest version of each submodule, and checking if it works. It gives us a heads up if a change in does not work well with the latest. For example, a correct validator change cause a glslang test to fail. Then we know we need to update the glslang test.

We expect this to work best if we have the latest for everything.

Comment (ben-clayton) [reply kokoro/scripts/linux/build-docker.sh:134]: Ack

**Pull request merged**